src/app/compiler/compiler-artefacts.js

```diff
@@ -32,6 +32,10 @@ module.exports = class CompilerArtefacts extends Plugin {
     this.on('lexon', 'compilationFinished', (file, source, languageVersion, data) => {
       this.compilersArtefacts['__last'] = new CompilerAbstract(languageVersion, data, source)
     })
+
+    this.on('yupl', 'compilationFinished', (file, source, languageVersion, data) => {
```

Comment:
is this a typo?

Reply:
indeed yes it is https://github.com/ethereum/remix-plugins-directory/blob/master/plugins/yulp/profile.json